server.js

```diff
@@ -333,3 +334,15 @@ app.get('/features/download/:type', (req, res) => {
         break;
     }
 });
+
+const ALL_CH_STR = ALL_CH.join(',');
+app.get('/features/client-hints/', (req, res) => {
```

Comment:
It would be nice to place this routes in the client hints directory, rather than continuing to expand the big server.js file. See how I did it for storage partitioning here :https://github.com/duckduckgo/privacy-test-pages/blob/main/privacy-protections/storage-partitioning/server/routes.js and here: https://github.com/duckduckgo/privacy-test-pages/blob/main/server.js#L48-L50

Reply:
Yep, 100%, in fact I'll do a follow up PR to clean up some of the old routes.